### PR TITLE
Trigger a scan of USB devices when loading integrations

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

- Called from the main integrations page and the onboarding
  integrations page

From https://github.com/home-assistant/core/pull/54904#discussion_r693135571

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/data/integration.ts [hunk 1]
```diff
@@ -52,3 +52,6 @@ export const fetchIntegrationManifest = (
 
 export const fetchIntegrationSetups = (hass: HomeAssistant) =>
   hass.callWS<IntegrationSetup[]>({ type: "integration/setup_info" });
+
+export const scanUSBDevices = (hass: HomeAssistant) =>
```
Comment (bramkragten) [inline src/data/integration.ts:56]: Should this be in integration? I guess it would make more sense as a separate file `data/usb`

Comment (bdraco) [reply src/data/integration.ts:56]: Will move it later today

Comment (bdraco) [reply src/data/integration.ts]: done

**Review by bramkragten — APPROVED** [review]
I assume the backend handles this as a no-op for OS's that don't need to be pulled for new devices?

**Comment by bdraco** [comment]
> I assume the backend handles this as a no-op for OS's that don't need to be pulled for new devices?

Yes and it has a debouncer built in to prevent excessive scans on OS that use scanning and don't have the observer.

**Pull request merged**